### PR TITLE
heimdall monitoring

### PR DESCRIPTION
**Pull request by ramakrishna183**
Enabling Prometheus to monitor Heimdall

### CODE REVIEW

File: monitoring/heimdall_monitoring.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#            3: Create Grafana Dashboard for Heimdall.		            #
+#									    #
+#############################################################################
+
+## path of Grafana datasorce and dashboard json's
+json_path=$(pwd)
+echo $json_path
+
+## Enabling Prometheus for Heimdall
+sudo sed -i 's/prometheus = false/prometheus = true/' /home/ubuntu/.heimdalld/config/config.toml
```
Comment (jdkanani) [inline monitoring/heimdall_monitoring.sh]: Can we use the home directory variable `$HOME` or something instead of using `home/ubuntu`?

Comment (ramakrishna183) [reply monitoring/heimdall_monitoring.sh]: @jdkanani ya we can use **$HOME** or **~/** so, what do you say ?

Comment (jdkanani) [reply monitoring/heimdall_monitoring.sh]: @ramakrishna183 Sounds good.

**Comment by vaibhavchellani** [comment]
Does this need to be in the Heimdall repo? Also please send PR's to develop first :)

**Pull request closed without merging**